### ISSUE TITLE
Black screen flickering - high refresh rate

### ISSUE THREAD

**Issue by OAKENSHIELD43** [StrangeLoopGames/EcoIssues#19818]
In my multiplayer game (based on a seperate server) I am encountering an issue where the screen flickers (and appears to re-render all objects) every few seconds or so. 
At higher framerates (100Hz) the effect is so potent the game is nearly unplayable.
When the framerate is manually reduced (disabling V-sync) to around 40Hz the effect is lessened, but still present.

Specifications:
Screen: Acer x34 (3440x1440p)
Graphics card: 1080Ti ROG
CPU: i7-7700k @3.5Ghz

I've not encountered this issue in previous versions of the game using the same hardware.

**Comment by OAKENSHIELD43** [comment]
Changing the settings to borderless or windowed has no effect on the issue. 
Moving around in the game world causes more flickering than not moving.

**Comment by OAKENSHIELD43** [comment]
Adjusting graphics settings to default or setting details to minimum has no effect on flickering

**Comment by OAKENSHIELD43** [comment]
Adjusting settings in Nvidea control panel has no effect, followed this guide here: https://blog.mivia.dk/solved-screen-flickering-and-screen-turning-black-playing-game/#fix-screen-flickering-nvidia-graphics 

Reinstalling the game and verifying game files had no effect on the flickering

**Comment by SlayksWood** [comment]
@OAKENSHIELD43 Thank you for the report. Can you make a video because I don't get what you mean with Black screen flickering.

**Comment by mov51** [comment]
I believe this is what they mean. It seems to happen in specific areas and correspond to the background sounds changing. 

https://youtu.be/MHgkdLg_HLE

**Comment by OAKENSHIELD43** [comment]
My apologies on not replying sooner, I seem to have fixed the worst of the
screen flickering, however it still happens in certain locations. I cannot
remember exactly how I fixed the flickering, however it had to do with
adjusting screen refresh rate in Nvidea control panel. My screen is now at
90Hz and it seems to have fixed the intermittent issues.
I ran the game windowed (not fullscreen) and even the taskbar was
flickering, suggesting an issue with V-sync (which I am using).

The video above is the exact issue I was experiencing, on flat ground but
where my campsite used to be placed.
